src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int johnSalary = 0;
+        int andrewSalary = 0;
+        int kateSalary = 0;
```

Comment:
The initialization of separate salary variables for 'John', 'Andrew', and 'Kate' is not scalable. Consider using a data structure like a `Map<String, Integer>` to store salaries for each employee dynamically.